document/core/appendix/properties.rst

```diff
@@ -66,9 +66,9 @@ Results
 
 * Let :math:`[t^\ast]\to[t_0^\ast]` be the :ref:`tag type <syntax-tagtype>` |tagtype|.
 
-* Assert: due to :ref:`validation <valid-externval-tag>` the type sequence :math:`t_0^\ast` must be empty.
+* Due to :ref:`validation <valid-externval-tag>` the type sequence :math:`t_0^\ast` must be empty.
```

Comment:
```suggestion
* The type sequence :math:`t_0^\ast` must be empty.
```
It's not due to some other validation, this rule _is_ the validation. ;)

Reply:
> Externval-tag validity, which you referenced here, does not actually check that the referenced tag instance is valid (and this rule does not know if the store is valid).

Thank you, I think I missed that point. I saw that [externval-tag validation](https://webassembly.github.io/exception-handling/core/valid/types.html#xref-syntax-types-syntax-tagtype-mathsf-tag-xref-syntax-types-syntax-tagtype-mathit-tagtype) does check if the tag is valid, and I forgot that it's not aware of the store's validity.

 I will implement your second suggestion to combine the first three bullets. This way it's shorter and consistent with THROWadm validation.

> Finally, I noticed that we are currently a bit inconsistent: tag type validity requires that the result is empty, but we are simply assuming, not asserting that in various places of the prose (e.g., the validation rule for THROW). My preference for fixing this would be to not treat tag types with empty results as invalid themselves, but only enforce emptiness in the exception-related use site rules.

I will create a new PR to make these changes, to not mix up the conversations.

Reply:
@rossberg 

> I will implement your second suggestion to combine the first three bullets. This way it's shorter and consistent with THROWadm validation.

Done.